### PR TITLE
test: e2e quickstart test case

### PR DESCRIPTION
**Pull request by JeyJeyGao**
Please follow this order to review: #493 -> this PR
Signed-off-by: Junjie Gao <junjiegao@microsoft.com>

### CODE REVIEW

File: test/e2e/suite/scenario/quickstart.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+			MatchKeyWords("notation/truststore/x509/ca/wabbit-networks.io/wabbit-networks.io.crt")
+	})
+
+	It("sign the container image", func() {
+		notation.Exec("sign", artifact.ReferenceWithDigest()).
+			MatchContent(fmt.Sprintf("Successfully signed %s\n", artifact.ReferenceWithDigest()))
+
+		notation.Exec("ls", artifact.ReferenceWithDigest()).
+			MatchKeyWords(fmt.Sprintf("%s\n└── application/vnd.cncf.notary.signature\n    └── sha256:", artifact.ReferenceWithDigest()))
+	})
```
Comment (shizhMSFT) [inline test/e2e/suite/scenario/quickstart.go:59]: In the quick start, there is
```
notation sign --signature-format cose $IMAGE
```
We also need to include that in the E2E test.

**Review by patrickzheng200 — APPROVED** [review]
LGTM

File: test/e2e/suite/scenario/quickstart.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+	It("Verify the container image", func() {
+		notation.Exec("verify", artifact.ReferenceWithDigest()).
+			MatchContent(fmt.Sprintf("Successfully verified signature for %s\n", artifact.ReferenceWithDigest()))
+	})
+
+	It("Verify the container image with cose format", func() {
+		notation.Exec("verify", artifact2.ReferenceWithDigest()).
+			MatchContent(fmt.Sprintf("Successfully verified signature for %s\n", artifact2.ReferenceWithDigest()))
+	})
```
Comment (shizhMSFT) [inline test/e2e/suite/scenario/quickstart.go:83]: This test cannot tell whether JWS fails or COSE fails. It can only tell both fails.

Comment (JeyJeyGao) [reply test/e2e/suite/scenario/quickstart.go]: As you can see,
```go
It("Verify the container image with cose format", func() {
```
This line explains what is doing and will be shown in the test output.
Also updated the description for verifying JWS signature, so the output can tell whether JWS or COSE is failed.

Comment (JeyJeyGao) [reply test/e2e/suite/scenario/quickstart.go]: example output
```
• [0.042 seconds]
notation quickstart E2E test
/home/jj/projects/junjie/notation/test/e2e/suite/scenario/quickstart.go:13
  Verify the container image with jws format
  /home/jj/projects/junjie/notation/test/e2e/suite/scenario/quickstart.go:75

  Begin Captured StdOut/StdErr Output >>

    >> should pass: /home/jj/projects/junjie/notation/bin/notation verify localhost:5000/e2e-8223469589761937864@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f >>
    Successfully verified signature for localhost:5000/e2e-8223469589761937864@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f
  << End Captured StdOut/StdErr Output

  Begin Captured GinkgoWriter Output >>
    STEP: 
    >> should pass: /home/jj/projects/junjie/notation/bin/notation verify localhost:5000/e2e-8223469589761937864@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f >> 01/16/23 16:50:07.164
  << End Captured GinkgoWriter Output
------------------------------
notation quickstart E2E test
  Verify the container image with cose format
  /home/jj/projects/junjie/notation/test/e2e/suite/scenario/quickstart.go:80
------------------------------
• [0.032 seconds]
notation quickstart E2E test
/home/jj/projects/junjie/notation/test/e2e/suite/scenario/quickstart.go:13
  Verify the container image with cose format
  /home/jj/projects/junjie/notation/test/e2e/suite/scenario/quickstart.go:80

  Begin Captured StdOut/StdErr Output >>

    >> should pass: /home/jj/projects/junjie/notation/bin/notation verify localhost:5000/e2e-15492302041286846102@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f >>
    Successfully verified signature for localhost:5000/e2e-15492302041286846102@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f
  << End Captured StdOut/StdErr Output

  Begin Captured GinkgoWriter Output >>
    STEP: 
    >> should pass: /home/jj/projects/junjie/notation/bin/notation verify localhost:5000/e2e-15492302041286846102@sha256:cc2ae4e91a31a77086edbdbf4711de48e5fa3ebdacad3403e61777a9e1a53b6f >> 01/16/23 16:50:07.207
  << End Captured GinkgoWriter Output

```

**Review by rgnote — APPROVED** [review]
LGTM

**Pull request merged**